routers/web/feed/profile.go

```diff
@@ -39,10 +41,22 @@ func showUserFeed(ctx *context.Context, formatType string) {
 		return
 	}
 
+	ctxUserDescription, err := markdown.RenderString(&markup.RenderContext{
```

Comment:
This is an unrelated change?

Reply:
The description: `render description for User/Org rss`

On web UI, it's also rendered as markdown IIRC.

Reply:
It's also a related rss bugfix ... but I can split it out

Reply:
Y/N ?

Reply:
-> #23887